src/arizona_live_view.erl

```diff
@@ -94,6 +90,13 @@ parse_str(Str, Macros) ->
 mount(Mod, Socket) ->
     Mod:mount(Socket).
 
+-spec render(Mod, Macros) -> Tree
+    when Mod :: module(),
+         Macros :: macros(),
+         Tree :: arizona_tpl_parse:tree().
+render(Mod, Macros) ->
+    Mod:render(Mod, Macros).
+
```

Comment:
Was not present. Is a helper as the other ones are. I think we're not using it specifically, because we do an `apply` (?).

Reply:
(yeah, I think it's because of that situation we discussed: this is coming from the config. but still not hard-coded)